src/CarbonAware.Aggregators/src/CarbonAware/ICarbonAwareAggregator.cs

```diff
@@ -13,11 +13,13 @@ public interface ICarbonAwareAggregator : IAggregator
     Task<IEnumerable<EmissionsData>> GetEmissionsDataAsync(IDictionary props);
 
     /// <summary>
-    /// Returns best emissions data record.
+    /// Returns best emissions data record(s). Multiple "best" records are returned in case
```

Comment:
Consider moving the `Multiple "best" records are returned...` comment to `<remarks>` rather than as part of the summary.